src/configuration.ts

```diff
@@ -12,16 +12,16 @@ export class Configuration {
 	public get<T> (section?: string, resource?: Uri, defaultValue?: T): T {
 		return defaultValue === undefined
 			? workspace
-				.getConfiguration(section === undefined ? undefined : ExtensionId, resource!)
-				.get<T>(section === undefined ? ExtensionId : section)!
+				.getConfiguration(section === undefined ? undefined : ExtensionName, resource!)
+				.get<T>(section === undefined ? ExtensionName : section)!
 			: workspace
-				.getConfiguration(section === undefined ? undefined : ExtensionId, resource!)
-				.get<T>(section === undefined ? ExtensionId : section, defaultValue)!;
+				.getConfiguration(section === undefined ? undefined : ExtensionName, resource!)
+				.get<T>(section === undefined ? ExtensionName : section, defaultValue)!;
 	}
 
 	public update (section: string, value: any, target: ConfigurationTarget, resourece?: Uri): Thenable<void> {
```

Comment:
<!--
  0 failure: 
  2 warning:  src/configuration..., src/configuration...
  
  
  DangerID: danger-id-Danger;
  File: src/configuration.ts;
  Line: 22;
-->

- :warning: src/configuration.ts line 22 – Argument 'value' should be typed with a non-any type. (@typescript-eslint/explicit-module-boundary-types)
- :warning: src/configuration.ts line 22 – Unexpected any. Specify a different type. (@typescript-eslint/no-explicit-any)